### PR TITLE
Updated lookup-table.md with CSV format requirements

### PR DESCRIPTION
**Pull request by akramarev**
# Amplitude Developer Docs PR

## Description

see https://amplitude.atlassian.net/browse/HC-551

## Deadline

not urgent

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [X] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by markzegarelli — APPROVED** [review]

File: docs/data/sources/lookup-table.md [hunk 1]
```diff
@@ -55,13 +61,13 @@ With Amplitude's Lookup Table feature, you can import your own data and map it t
 - **Bulk filter long lists**. You want to see user behavior for a specific region and you have a list of all the customers and their regions. Use this to map each customer to a region, creating a new "Region" property. Now you can filter specifically to each region in a chart.
 
 ## CSV example
-The following CSV will create a new property called `Language` that maps to the `SKU` property. This CSV is an example implementation of the **Bulk update property values** use case from the previous section.
 
+The following CSV will create a new property called `Language` that maps to the `SKU` property. This CSV is an example implementation of the **Bulk update property values** use case from the previous section.
```
Comment (markzegarelli) [inline docs/data/sources/lookup-table.md:65]:
```suggestion
The following CSV creates a new property called `Language` that maps to the `SKU` property. This CSV is an example implementation of the **Bulk update property values** use case from the previous section.
```

File: docs/analytics/apis/lookup-tables-api.md [hunk 2]
```diff
@@ -25,7 +25,14 @@ You can also create and manage lookup tables from the Amplitude web app. See [Lo
 
 ## Considerations
 
-The max file size is 100 MB and the file can't have more than 1,000,000 rows.
+The CSV file must comply with the following requirements:
+
+- The max file size is 100 MB and the file can't have more than 1,000,000 rows.
+- The first row should contain column names/headers.
```
Comment (markzegarelli) [inline docs/analytics/apis/lookup-tables-api.md:31]: Is this a requirement or a recommendation? Will the import fail if no header row is present?

File: docs/analytics/apis/lookup-tables-api.md [hunk 3]
```diff
@@ -25,7 +25,14 @@ You can also create and manage lookup tables from the Amplitude web app. See [Lo
...
+- The first column must correspond to the mapping property value and must contain *unique* values. Keep in mind that Lookup Tables are looking for exact matches, and are *case-sensitive*.
```
Comment (markzegarelli) [inline docs/analytics/apis/lookup-tables-api.md:32]:
```suggestion
- The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/analytics/apis/lookup-tables-api.md (hunk 2 above)
Comment (akramarev) [reply docs/analytics/apis/lookup-tables-api.md]: it's a requirement, updating..

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**